src/test/skript/tests/syntaxes/conditions/CondIsWithin.sk

```diff
@@ -31,3 +31,11 @@ test "within condition":
 	assert {_loc1} is not within {_pig} with "failed within entity"
 
 	delete random entity of {_pig}
+
+test "within multiple condition":
```

Comment:
This test file seems to miss out on testing blocks with multiple bounding boxes
A test for a point in the upper part of a stair would be good.